models/articles.models.js

```diff
@@ -1,10 +1,24 @@
 const db = require('../db/connection');
 
+exports.checkArticleIdExists = (article_id) => {
+    if (/\D+/.test(article_id)) {
```

Comment:
We could let psql deal with this error, as it's invalid it should throw an error in the query.

Reply:
should I just remove it, then?